networking_cisco/plugins/cisco/device_manager/plugging_drivers/aci_vlan_trunking_driver.py

```diff
@@ -123,38 +122,34 @@ def get_vrf_context(self):
         else:
             return None
 
-    @property
-    def get_vrf_details(self):
-        if self.apic_driver:
-            return self._get_vrf_details
-        else:
-            return None
-
     def _get_vrf_context_gbp(self, context, router_id, port_db):
         l2p = self.apic_driver._network_id_to_l2p(
             context, port_db['network_id'])
+        # TODO(tbachman): should we just use the l3_policy_id
```

Comment:
I see. then we should definitely just use l2p['l3_policy_id'] directly if that info is already there.